lib/paper_trail/has_paper_trail.rb

```diff
@@ -57,6 +57,41 @@ def has_paper_trail(options = {})
         setup_model_for_paper_trail(options)
 
         setup_callbacks_from_options options[:on]
+
+        setup_callbacks_for_habtm options[:join_tables]
+      end
+
+      def update_for_callback(name, callback, model, assoc)
+        paper_trail_habtm = model.instance_variable_get(:@paper_trail_habtm) || {}
```

Comment:
Is there any way we can avoid using `instance_variable_get`? Could we, for example, add an instance accessor method?